### PR TITLE
Changing all OKD instances to OCP

### PR DESCRIPTION
**Pull request by fwji**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -8,18 +8,18 @@ permalink: /guides/app-logging/
...
-:page-description: Learn how to do application logging with Kabanero on OKD with Elasticsearch, Fluentd, and Kibana.
+:page-description: Learn how to do application logging with Kabanero on OCP with Elasticsearch, Fluentd, and Kibana.
 :guide-author: Kabanero
 :page-tags: ['logging', 'Elasticsearch', 'Fluentd', 'Kibana']
 :repo-description: Visit the https://kabanero.io/guides/{projectid}.html[website] for the rendered version of the guide.
-= Application Logging with Kabanero on OKD with Elasticsearch, FluentD, and Kibana
+= Application Logging with Kabanero on OCP 4.2 with Elasticsearch, FluentD, and Kibana
 
-__The following guide has been tested with OKD 4.2/Kabanero 0.3.0.__
+__The following guide has been tested with OCP 4.2/Kabanero 0.3.0rc1.__
```
[the remark this answers is not in the document]
Comment (donbourne) [reply README.adoc]: OCP -> RHOCP

File: README.adoc [hunk 2]
```diff
@@ -8,18 +8,18 @@ permalink: /guides/app-logging/
 :page-guide-category: basic
 :page-essential: true
 :page-essential-order: 1
-:page-description: Learn how to do application logging with Kabanero on OKD with Elasticsearch, Fluentd, and Kibana.
+:page-description: Learn how to do application logging with Kabanero on RHOCP with Elasticsearch, Fluentd, and Kibana.
 :guide-author: Kabanero
 :page-tags: ['logging', 'Elasticsearch', 'Fluentd', 'Kibana']
 :repo-description: Visit the https://kabanero.io/guides/{projectid}.html[website] for the rendered version of the guide.
-= Application Logging with Kabanero on OKD with Elasticsearch, FluentD, and Kibana
+= Application Logging with Kabanero on Red Hat Openshift Container Platform 4.2 with Elasticsearch, Fluentd, and Kibana
```
Comment (donbourne) [inline README.adoc]: Openshift -> OpenShift

File: README.adoc [hunk 3]
```diff
@@ -8,18 +8,18 @@ permalink: /guides/app-logging/
...
 
-__The following guide has been tested with OKD 4.2/Kabanero 0.3.0.__
+__The following guide has been tested with OCP 4.2/Kabanero 0.3.0.__
```
Comment (donbourne) [inline README.adoc]: OCP -> Red Hat OpenShift Container Platform (RHOCP)

**Comment by fwji** [comment]
Preview of the published guide can be visited here http://rhel7-openshift.rtp.raleigh.ibm.com:4000/guides/app-logging/#install-cluster-logging

**Pull request merged**